Grizzlies-SpyDuh/Repository/UserRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Grizzlies_SpyDuh.Utils;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Hosting;
+using System.Collections.Generic;
+
+namespace Grizzlies_SpyDuh.Repository
+{
+    public class UserRepository //: BaseRepository, IUserRepository
+    {
+        /*-------------------GetBySkill()----------------------*/
+        public User GetBySkill(string skillName)
```

Comment:
This Class isn't using the correct namespace for access to user. Should add a using statement to fix this at the top of the file.